terraform/firewall/test/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "id" {
+  value = module.firewall.id
+}
```

Comment:
Add

output "name" {
  value = module.firewall.name
}
output "resource_group_name" {
  value = module.firewall.resource_group_name
}

Reply:
Just implemented suggested changes. Resolving conversation. Re-open conversation if you have further suggestions or disagree with the changes proposed and implemented.